src/ts/components/dates/DatePickerInput.tsx

```diff
@@ -101,6 +101,7 @@ const DatePickerInput = (props: Props) => {
                 minDate={stringToDate(minDate)}
                 maxDate={stringToDate(maxDate)}
                 excludeDate={isExcluded}
+                popoverProps={{returnFocus: true}}
                 {...others}
```

Comment:
Would we want to merge this with any `popoverProps` you provide explicitly? ie pull `popoverProps` out of `props`, then here:
```
popoverProps={{returnFocus: true, ...popoverProps}}
```

Reply:
oh nice catch - yes, that's how it should work.

Reply:
That needs to be applied to #471 too